src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/commonmodels/Address.java

```diff
@@ -0,0 +1,146 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class Address {
```

Comment:
Some more fields than in ODP but can align later: 
 houseNameNum: string, // property_name_number
           street: string, // line_1
           area: string, // line_2 
           postTown: // locality
           region: // county, region
           country:. // country 
           postCode: // postcode